iron-dropdown.html

```diff
@@ -271,8 +272,7 @@
         },
 
         _cancelAnimations: function() {
-          this.cancelAnimation('open');
-          this.cancelAnimation('close');
+          this.cancelAnimation();
```

Comment:
This is interesting, the playAnimation method takes an argument but cancelAnimation method doesn't: https://github.com/PolymerElements/neon-animation/blob/master/neon-animation-runner-behavior.html#L107